assets/src/editor/events/ticket-assignments-manager/submit-ticket-assignments-button.js

```diff
@@ -37,11 +40,31 @@ const SubmitTicketAssignmentsButton = ( {
 		beforeEditorClose,
 		closeEditor,
 	} );
+	const onClick = async () => {
+		const { forPersist, id, isNew, modelName } = dateEntity;
+		const path = applyQueryString( modelName );
+		const request = {
+			path: isNew ? path : path + '/' + id,
+			method: isNew ? 'POST' : 'PUT',
+			data: forPersist,
+		};
+
+		try {
+			processChanges();
+			await apiFetch( request );
+		} catch ( e ) {
+			/**
+			 * handle error with smth like:
+			 * https://github.com/WordPress/gutenberg/tree/master/packages/components/src/notice
+			 */
+		}
+	};
+
 	return useMemo( () => (
 		<FormSubmitButton
-			onClick={ () => processChanges() }
+			onClick={ onClick }
 			buttonText={ __(
-				'Update Ticket Assignments',
+				'Updating Ticket Assignments AND CREATE EVENT DATE NOW !!!',
```

Comment:
srsly? way too much text for a button and what's with the all caps?

Reply:
:)
it is just a draft PR and it will get closed, it wasn't intended to be merged